crates/net/eth-wire/src/types/blocks.rs

```diff
@@ -43,11 +43,11 @@ pub struct GetBlockHeaders {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct BlockHeaders(
```

Comment:
I think this type can be deduped? I.e. instead of using `BlockHeaders` we can just use `Headers`

Reply:
Deduped this but it introduced a bunch of semantic changes in the `eth-wire`. Specifcally in `EthMessage` and `PeerResponse`.